src/lib/ngsi/MetadataVector.cpp

```diff
@@ -87,6 +87,25 @@ std::string MetadataVector::render(const std::string& indent, bool comma)
 
 /* ****************************************************************************
 *
+* MetadataVector::matchFilter -
+*
+*/
+bool MetadataVector::matchFilter(const std::string& mdName, const std::vector<std::string>& metadataFilter)
+{
+
+  /* Metadata filtering only in the case of actual metadata vector not conatining "*" */
+  if ((metadataFilter.size() == 0) || (std::find(metadataFilter.begin(), metadataFilter.end(), NGSI_MD_ALL) != metadataFilter.end()))
+  {
+    return true;
+  }
```

Comment:
What about the two special metadata?
It seems here that TRUE is returned if metadataFilter is empty or contains "*",
but I understand this is not enough for those two, they must be explicitly there ...

Right?